portal-web/test/functional/com/liferay/portalweb/tests/enduser/collaboration/blogs/PGBlogs.testcase

```diff
@@ -1393,6 +1393,24 @@ definition {
 			sender = "Joe Bloggs",
 			subject = "Blogs Entry Title"
 		);
+
+		Navigator.gotoPage(pageName = "Blogs Page");
+
+		Blogs.editEntryPG(
+			entryContent = "Blogs Entry Content",
+			entryContentEdit = "Blogs Entry Content Edit",
+			entryTitle = "Blogs Entry Title",
+			entryTitleEdit = "Blogs Entry Title Edit"
```

Comment:
@Hanlf you are missing a comma at the end of this line

Reply:
Apologize for the inconvenience caused, thank you @jpince for catching it, sorry!

Reply:
@Hanlf no problem, we are all still learning the new POSHI Script :)